doc/docportal/help/report_bugs.rst

```diff
@@ -28,28 +28,24 @@ The ScummVM log file
 To help you report a bug, you can find error messages in the ScummVM log file. The location of this file varies depending on your operating system.
 
 
-.. tabbed:: macOS
+.. tabbed:: Windows
```

Comment:
The diff on this file is difficult to follow, but basically it's just that the Windows tab is first, matching the other pages.